src/core/xref.js

```diff
@@ -172,6 +172,7 @@ function getRequestEntry(elem) {
   const forContext = getForContext(elem, isIDL);
 
   return {
+    id: "", // TBD
```

Comment:
To ensure the shape of object returned is same (minor optimization, and also to make TypeScript happier).

Reply:
Should add this as a comment instead 😄 
```suggestion
    // Add an empty `id` to ensure the shape of object returned is same (minor optimization, and also makes TypeScript happy).
    id: "",
```